shell/ci/release/docker.sh

```diff
@@ -33,7 +33,8 @@ fi
 get_image_field() {
   local name="$1"
   local field="$2"
-  yq -r ".[\"$name\"]$field" "$MANIFEST"
+  # shellcheck disable=SC1087 # this is a yq/jq filter that shellcheck thinks is bash
+  yq -r ".[\"$name\"]$field[]" "$MANIFEST"
```

Comment:
hilarious, of course it was something that small

Reply:
lol pretty much. yq/jq syntax is crazy though